src/pudl/package_data/glue/ferc1_eia_train.csv

```diff
@@ -1,213 +1,6 @@
 record_id_eia,record_id_ferc1,signature_1,signature_2,notes
```

Comment:
Hm, no. The 1:m process does not overwrite this CSV, but merely updates the records internally. Any changes here are a result of running the validation process with the complete set of newly integrated data. If this shouldn't be included in this PR, I can remove it (though I'm not sure why the result of this would be different than what's currently in `dev`).

Reply:
That's what I'm curious about. It might be a good idea to compare this file to the one in dev to see what's going on. I don't see why there would be any changes. One thought is that the new training data might count as overrides even if it's already in the old training data. i.e., it replaces current data with the same values instead of only replacing values that are different.

Reply:
Some look exactly the same, some have different signatures, and some only appear in the new data. Which suggests to me that the source data from your PR and this one aren't actually the same.